extensions/roc-package-web-app-react/src/config/roc.config.js

```diff
@@ -26,7 +26,7 @@ export default {
             ssr: true,
             template: {
                 path: undefined,
-                name: 'main.html',
+                name: 'roc-base-template.njk',
```

Comment:
I would like to keep the default template name for now since this otherwise would create breaking changes in a lot of projects that only have changed the template directory but not the template name.